php/commands/core.php

```diff
@@ -535,7 +535,7 @@ function wp_new_blog_notification() {
 			'title' => '',
 			'admin_user' => '',
 			'admin_email' => '',
-			'admin_password' => ''
+			'admin_password' => wp_generate_password(18)
```

Comment:
This approach will cause the admin password to never display. See later on:

```
if ( empty( $admin_password ) ) {
 			WP_CLI::log( "Admin password: {$result['password']}" );
}
```

The expected behavior is that `Admin password:` displays if no password is provided.

You'll need to modify your logic to meet this expected behavior. If this doesn't fail tests, you should also add a test for this.